server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -730,7 +730,9 @@ public void apply(Settings value, Settings current, Settings previous) {
 
                 RemoteStoreSettings.CLUSTER_REMOTE_INDEX_SEGMENT_METADATA_RETENTION_MAX_COUNT_SETTING,
                 RemoteStoreSettings.CLUSTER_REMOTE_TRANSLOG_BUFFER_INTERVAL_SETTING,
-                RemoteStoreSettings.CLUSTER_REMOTE_TRANSLOG_TRANSFER_TIMEOUT_SETTING
+                RemoteStoreSettings.CLUSTER_REMOTE_TRANSLOG_TRANSFER_TIMEOUT_SETTING,
+
+                SearchService.MAX_AGGREGATION_REWRITE_FILTERS
```

Comment:
Can you move this up to around line 522-527 (i.e. with the other `SearchService` settings)?

Reply:
My bad, did look odd to me. Thanks for pointing out @msfroh!

Reply:
Updated